Makefile

```
@@ -266,6 +266,8 @@ artifacts: kustomize yq helm ## Generate release artifacts.
 	$(KUSTOMIZE) build config/alpha-enabled -o artifacts/manifests-alpha-enabled.yaml
 	$(KUSTOMIZE) build config/prometheus -o artifacts/prometheus.yaml
 	$(KUSTOMIZE) build config/visibility -o artifacts/visibility-api.yaml
+	$(KUSTOMIZE) build config/apf/default -o artifacts/apf.yaml
+	$(KUSTOMIZE) build config/apf/1_28 -o artifacts/apf-1-28.yaml
```

```suggestion
	$(KUSTOMIZE) build config/apf/default -o artifacts/visibility-apf.yaml
	$(KUSTOMIZE) build config/apf/1_28 -o artifacts/visibility-apf-1-28.yaml
```
so that the names are more descriptive, since the artifact names are user-facing. Just `apf.yaml` may suggest the scope of the config is wider.

---

Done.

---

Moved apf to the `config/visibility-apf` directory because Helm generates templates by visibility directory and includes APF too.